src/ctl/src/lib.rs

```diff
@@ -231,6 +231,19 @@ enum ScaleCommands {
     /// The resize command scales up and down the cluster by specifying the worker ids to be
     /// included and excluded.
     Resize(ScaleResizeCommands),
+    /// mark a compute node as unschedulable
+    #[clap(verbatim_doc_comment)]
+    Cordon {
+        /// IDs of the compute node to cordon. List the IDs via cluster-info
+        #[clap(long, value_delimiter = ',', value_name = "id,...")]
+        workers: Vec<u32>,
```

Comment:
As a user, how can I get the worker_id in my cluster? In particular, I need to know the mapping between Kubernetes pods and the worker id here, so that I can `cordon` the right node before shutting it down in K8s.

I just saw the demo screen recording and it was using the `Host:IP` string instead of worker ID. 👀

Reply:
That is correct. I originally used the `Host:IP`, but changed in this PR to the worker ID in accordance with the [design document](https://www.notion.so/risingwave-labs/High-Level-scaling-interface-for-risectl-2aa34dc3b253431c8a13c0fb803ea94e) by @shanicky. 

You can retrieve the worker ID by running `risedev ctl meta cluster-info`

Example output: 
```
+------------------+---------------+
| Compute Node     | Parallel Unit |
+==================================+
| 1@127.0.0.1:5687 | 0             |
|------------------+---------------|
|                  | 1             |
|------------------+---------------|
|                  | 2             |
|------------------+---------------|
|                  | 3             |
|------------------+---------------|
| 2@127.0.0.1:5688 | 4             |
|------------------+---------------|
|                  | 5             |
|------------------+---------------|
...
```

Here worker 1 is running at port `5687`.

Reply:
I prefer to use `Host:IP` for risedev ctl command, and put the id resolve logic inside the `cordon` command.